build.gradle

```diff
@@ -315,10 +315,13 @@ task(packageSageJar)  {
  *
  * @return build number
  */
-def getBuildNumber() {
-  def buildVerText = new File('java/sage/SageConstants.java').text
+def getBuildNumber() 
+{
+  def buildVer
```

Comment:
Erase this line and the next line and put the 'def' right before buildVer a coouple lines down.